### PR TITLE
Update box.json

### PR DESCRIPTION
**Pull request by tcowles**
Updated Description, Top KeyWords and Contract Vehicle

### CODE REVIEW

File: _data/products/box.json [hunk 1]
```diff
@@ -8,19 +8,24 @@
         "file sharing",
         "flash storage",
         "cloud computing",
-        "saas"
+        "saas",
+        "enterprise software",
+        "content management",
+        "productivity",
+        "mobile",
+        "ECM"
```
Comment (stroupaloop) [inline _data/products/box.json]: please only select up to 5 keywords

**Comment by stroupaloop** [comment]
need to trim down the top_keywords to five, otherwise it looks good!

**Comment by tcowles** [comment]
Hey Andrew - trimmed it down to 5.

Also - Site looks great!

Thanks,
Tom

_Tom Cowles_
Senior Compliance Manager
301-651-0636

**Comment by stroupaloop** [comment]
looks great, adding it in now!

**Pull request merged**